### PR TITLE
made missing value handling method

### PR DESCRIPTION
**Pull request by Cooksen**

### CODE REVIEW

File: decanter_ai_sdk/client.py [hunk 1]
```diff
@@ -121,6 +121,7 @@ def train_iid(
         seed: int = 1180,
         timeseries_value: List[Dict[str, Any]] = [],
         holdout_percentage: int = 10,
+        missing_value_settings: Dict[str, Any] = {},
```
Comment (jwolf9) [inline decanter_ai_sdk/client.py:124]: think it would be better if the values were an enum

File: decanter_ai_sdk/client.py [hunk 2]
```diff
@@ -271,6 +284,7 @@ def train_ts(
         seed: int = 1111,
         drop_features: List[str] = [],
         custom_feature_types: Dict[str, DataType] = {},
+        missing_value_settings: Dict[str, Any] = {},
```
Comment (jwolf9) [inline decanter_ai_sdk/client.py:287]: same

**Review by jwolf9 — CHANGES_REQUESTED** [review]

File: decanter_ai_sdk/client.py [hunk 3]
```diff
@@ -121,7 +123,7 @@ def train_iid(
         seed: int = 1180,
         timeseries_value: List[Dict[str, Any]] = [],
         holdout_percentage: int = 10,
-        missing_value_settings: Dict[str, Any] = {},
+        missing_value_settings: Dict[str, Missing_Value_Handling] = {},
```
Comment (jwolf9) [inline decanter_ai_sdk/client.py:126]: Should not be a mixture of snake and camel case. Camel case is fine

File: decanter_ai_sdk/enums/missing_value_handling.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    Numerical missing value handling.
+        - Automatic
+        - Mean
+        - Median
+        - Zero
+        - DropRows
+    """
+
+    Automatic = "automatic"
+    Mean = "mean"
```
Comment (jwolf9) [inline decanter_ai_sdk/enums/missing_value_handling.py:15]: These can be all lower case

**Review by jwolf9 — CHANGES_REQUESTED** [review]

File: decanter_ai_sdk/client.py (hunk 3 above)
Comment (Cooksen) [reply decanter_ai_sdk/client.py:126]: OMG my bad!!

**Pull request merged**